test/sharness/t0290-cid.sh

```diff
@@ -128,7 +128,7 @@ cat <<EOF > codecs_expect
   151  eth-account-snapshot
   152  eth-storage-trie
   153  eth-receipt-log-trie
-  154  eth-reciept-log
+  154  eth-receipt-log
```

Comment:
This change will cause the test to fail. This is not a spelling error in the test, but rather one in the [`go-multicodec`](https://github.com/multiformats/go-multicodec) package. It was fixed [here](https://github.com/multiformats/multicodec/pull/326), but the fix is not yet a tagged version of  `go-multicodec`. 

Please run tests before submitting changes to the test code.

Reply:
OK, I will revert it.